ideagen3D ToughPLA.xml.fdm_material

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura" version="1.3">
+  <metadata>
+    <name>
+      <brand>ideagen3D</brand>
+      <material>PLA+</material>
```

Comment:
```suggestion
      <material>Tough PLA</material>
```

Check generic_tough_pla.xml.fdm_material; We use "Tough PLA" as the material type